github/misc.go

```diff
@@ -83,6 +83,61 @@ func (c *Client) ListEmojis(ctx context.Context) (map[string]string, *Response,
 	return emoji, resp, nil
 }
 
+// Conduct represents metric that contains name, key, url, body.
+type Conduct struct {
+	Name *string `json:"name,omitempty"`
+	Key  *string `json:"key,omitempty"`
+	URL  *string `json:"url,omitempty"`
+	BODY *string `json:"body,omitempty"`
+}
+
+func (c *Conduct) String() string {
+	return Stringify(c)
+}
+
+// ListCodesOfConduct returns all codes of conduct.
+//
+// https://developer.github.com/v3/codes_of_conduct/
```

Comment:
For consistency with our other endpoints, can you add a "GitHub API docs:" prefix to all the documentation URLs? Like so:

```Go
// GitHub API docs: https://developer.github.com/v3/codes_of_conduct/
```

Here, and in the method below where it's missing too.